### DISCUSSION TITLE
external plugin code snippets

### DISCUSSION THREAD

**Discussion by 18bitscom** [flyntwp/flynt#384] [Q&A] [answered]
Hello @steffenbew , @domtra ,

is there a way to display a code snippet like 

`[ess_grid alias="test_grid"]`

e.g. from another plugin like essential grid? In the normal classic text editor or gutenberg this works, but through the theme this is blocked. i.e. the goal would be to display the grid in the main content but the menu and footer should still be on the landingpage.

Looking forward to creative input.

**Comment by aaronmeder** [comment, marked as answer]
Hey @18bitscom 

I always make a simple Shortcode page component that renders a given shortcode.

I've just uploaded it to a new repo if you want to use it: https://github.com/aaronmeder/flynt-components
No magic included 😉

Aaron

**Reply by 18bitscom** [reply]
Hi @aaronmeder 

thanks in advance but your links shows a 404.
Where i can find the component?

Regards,
Hagrid

**Reply by aaronmeder** [reply]
Sorry the repo was still private - the link works now.

**Reply by 18bitscom** [reply]
Hello @aaronmeder,

awesome this works perfect with the Mailchimp snipped like MC4WP but not with essential grid code and i dont know why.
Is there something in the theme which blocks the essential grid scripts?

Hagrid

**Reply by 18bitscom** [reply]
Hey @aaronmeder,

the plugin developers of Essential Grid think that the grid is blocked by the theme. They have checked it via FTP. What needs to be done so that the shortcode can still be displayed?

Thanks in advance.

**Reply by steffenbew** [reply]
Hey @18bitscom, in our components when rendering content from WYSIWYGs, we're often filtering the HTML output for extra security with the `wp_kses_post` filter, like in the [BlockWysiwyg/index.twig template file](https://github.com/flyntwp/flynt/blob/master/Components/BlockWysiwyg/index.twig#L4).
Maybe this is also applied to your scenario and escapes markup from the plugin you're trying to use?
